cmd/nomos/status/client.go

```diff
@@ -354,6 +358,18 @@ func (c *ClusterClient) namespaceRepoClusterStatus(ctx context.Context, cs *Clus
 	}
 }
 
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and root-reconciler deployment
+// If root-reconciler exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func (c *ClusterClient) IsOssInstallation(ctx context.Context) bool {
+	v, _ := c.ConfigManagement.Version(ctx)
+	_, operatorDeployment := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, ACMOperatorDeployment, metav1.GetOptions{})
+	_, rootDeployment := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, "root-reconciler", metav1.GetOptions{})
```

Comment:
I don't think `root-reconciler` necessarily needs to exist, I would suggest checking either for the RSync CRDs or for the existence of `reconciler-manager`

Reply:
+1. We need to check the RSync CRD as you proposed, not the reconciler.

Reply:
Updated validation to check for RootSync CRD